eng/tox/tox.ini

```
@@ -12,6 +12,7 @@ deps =
 source =
     azure
     **/azure
+    opentelemetry
```

the package is going to be renamed to `microsoft.opentelemetry.exporter.azuremonitor` very very soon. 
```suggestion
    microsoft
```